XamlControlsGallery/PageHeader.xaml

```diff
@@ -80,7 +80,7 @@
                         Click="OnThemeButtonClick"
                         Label="Toggle Theme">
                         <AppBarButton.Icon>
-                            <FontIcon Glyph="&#xE771;" Margin="0,-2,0,0" />
+                            <FontIcon Glyph="&#xE771;" Margin="0,-4,0,0" />
```

Comment:
I'm always suspicious of negative margins and worry about touching this one. Is this change necessary?